aten/src/ATen/test/basic.cpp

```diff
@@ -2,6 +2,7 @@
 
 #include <ATen/ATen.h>
 #include <ATen/core/Reduction.h>
+#include <torch/cuda.h>
```

Comment:
what do you need from this?

Reply:
for `torch::cuda::is_available()`

Reply:
can you follow what the other tests in this directory do?  Namely, use `at::cuda::is_available()` (and don't include this file).

Reply:
I cant use `at::cuda::is_available()` here as i will get an error: `error: ‘is_available’ is not a member of ‘at::cuda’`. Other files that use `at::cuda::is_available()` are including `<ATen/cuda/CUDAContext.h>` that enables this. I picked `<torch/cuda.h>` as its much easier to add than `<ATen/cuda/CUDAContext.h>` which will require additional work.
I can leave it as it is or move the test to another file, possibly CUDA test where its already enabled. What do you prefer?

Reply:
why does that involve additional work?  You should do what the other tests do here or if your way is better, change the other tests to do things the new way.  Not have two ways of doing the same thing.